packages/shared/lib/participation/account.ts

```diff
@@ -93,6 +94,21 @@ export const partiallyUnstakedAmount: Readable<number> = derived(
     }
 )
 
+export const currentAccountTreasuryVoteValue = derived(
```

Comment:
Okay as a quick fix but we'll need to pass details of which vote to make it more generic